airflow/providers/amazon/aws/operators/bedrock.py

```diff
@@ -664,3 +669,198 @@ def execute(self, context: Context) -> str:
             )
 
         return ingestion_job_id
+
+
+class BedrockRaGOperator(AwsBaseOperator[BedrockAgentRuntimeHook]):
+    """
+    Query a knowledge base and generate responses based on the retrieved results with sources citations.
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:BedrockRaGOperator`
+
+    :param input: The query to be made to the knowledge base. (templated)
+    :param source_type: The type of resource that is queried by the request. (templated)
+        Must be one of 'KNOWLEDGE_BASE' or 'EXTERNAL_SOURCES', and the appropriate config values must also be provided.
+        If set to 'KNOWLEDGE_BASE' then `knowledge_base_id` must be provided, and `vector_search_config` may be.
+        If set to `EXTERNAL_SOURCES` then `sources` must also be provided.
+    :param model_arn: The ARN of the foundation model used to generate a response. (templated)
+    :param prompt_template: The template for the prompt that's sent to the model for response generation.
+        You can include prompt placeholders, which become replaced before the prompt is sent to the model
```

Comment:
```suggestion
        You can include prompt placeholders, which are replaced before the prompt is sent to the model
```

Reply:
Heh.  that's copypasta from their docs, but I agree.  I'll make the change here.